source/content/guides/integrated-composer/02-dependencies.md

```diff
@@ -5,14 +5,29 @@ description: Learn how to add or remove an individual site dependency.
 tags: [composer, workflow]
 categories: [get-started]
 contributors: [ari, edwardangert]
-reviewed: "2022-04-28"
 layout: guide
 showtoc: true
 permalink: docs/guides/integrated-composer/dependencies
 anchorid: dependencies
 ---
 
-This sections provides information on how to add or remove individual site dependencies.
+This sections provides information on dependency requirements and how to add or remove individual site dependencies.
+
+## Composer `require` and `require dev` Sections
+
+It's important to understand how `require` and `require dev` are used on the Pantheon platform.
+
+### Composer `require` Section
+
+You should use the `require` section of your `composer.json` file for dependencies your web project needs, even dependencies only used on non-Live environments. Drupal modules/themes and WordPress plugins/themes should always be in the `require` section, not the `require-dev` section. All dependencies in the `require` section are pushed to Pantheon Dev and Multidev environments. 
+
+You can use  `--no-dev` to prevent specific dependencies from being pushed to your your Dev and Multidev environments.
```

Comment:
You really don't have control over this. It will always be like this:
- dev dependencies pushed to dev/multidev
- dev dependencies NOT pushed to test/live